misc/dist/ios_xcode/export_options.plist

```diff
@@ -4,7 +4,17 @@
 <dict>
     <key>method</key>
     <string>$export_method</string>
+
     <key>teamID</key>
     <string>$team_id</string>
+
+    <key>provisioningProfiles</key>
+    <dict>
+        <key>$identifier</key>Ï
```

Comment:
Is this weird `Ï` character supposed to be here?

Reply:
Yeah, not sure how it got there, removed now, thank you.